docs/components.md

```diff
@@ -2,3 +2,6 @@
 
 [![K8GB Internal Components](/docs/images/k8gb-components.svg)](/docs/images/k8gb-components.svg)
 *The flow between internal k8gb components including scenario of Route53 integration.*
+
+[![K8GB Cluster Collaboration](/docs/images/k8gb-collaboration.svg)](/docs/images/k8gb-collaboration.svg) 
```

Comment:
@kuritka not sure if "cluster collaboration" is the right term here, maybe "multi-cluster interoperability"?